web/styles/modal.css

```diff
@@ -467,3 +467,19 @@
         }
     }
 }
+/* Styles for the checkbox UI grid */
```

Comment:
Instead of this comment, I'd wrap all these rules so that they start with the id of the modal. Based on my renaming of classes and ids, here's how I'd format these with a small margin adjustment for the add/remove all buttons flexbox ...

```scss
#generate-integration-url-modal {
    #integration-events-container {
        .integration-all-events-buttons {
            display: flex;
            gap: 10px;
            margin: 5px 0 10px;
        }

        .integration-event-wrapper {
            margin: 5px 0;
        }

        .integration-event-name {
            word-break: break-all;
        }
    }
}
```

Reply:
I think `.integration-event-name` should come under `#integrations-event-options`.